### ISSUE TITLE
[Feature Request] None throwing connectivity-check

### ISSUE THREAD

**Issue by WhatzGames** [neo4j/neo4j-dotnet-driver#690]
I have a case where I do not care for the reason of connectivity issues. I just need to know whether or not i am able to connect, that's it.
Maybe another option, as shown below could be a plausible option?

```csharp
namespace Neo4j.Driver;

public interface IDriver
{
    public Task<bool> TryVerifyConnectivityAsync();
}
```

**Comment by thelonelyvulpes** [comment]
@WhatzGames Thank you for you suggestion, I have had this on my personal backlog to implement for good while. I think what we will do is something like `Task<bool> TryGetServerInfoAsync(out IServerInfo)`

**Comment by WhatzGames** [comment]
@thelonelyvulpes That would be nice. If i recall it correctly, asynchronous operations  do not allow out parameters

**Comment by thelonelyvulpes** [comment]
Will be available in 5.7+ which should be available next month.
